### PR TITLE
Fixed typo error.

### PR DESCRIPTION
**Pull request by Suryadevelops**
Fixed some typo errors in the following lines

##Fixes
1. Line 166: Removed the and added that as the doesn't mean anything.

### CODE REVIEW

**Review by Edweis — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Features (things that add new or improved functionality to TypeScript) may be ac
 
 ## Legal
 
-You will need to complete a Contributor License Agreement (CLA). Briefly, this agreement testifies that you are granting us permission to use the submitted change according to the terms of the project's license, and that the work being submitted is under appropriate copyright. Upon submitting a pull request, you will automatically be given instructions on how to sign the CLA.
+You will need to complete a Contributor License Agreement (CLA). Briefly, this agreement testifies that you are granting us permission to use the submitted change according to the terms of the project's license and that the work being submitted is under appropriate copyright. Upon submitting a pull request, you will automatically be given instructions on how to sign the CLA.
```
Comment (RyanCavanaugh) [inline CONTRIBUTING.md:91]: Please revert. This isn't a typo.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ Your pull request should:
 * Include a description of what your change intends to do
 * Be based on reasonably recent commit in the **master** branch
 * Include adequate tests
-    * At least one test should fail in the absence of your non-test code changes. If your PR does not match this criteria, please specify why
+    * At least one test should fail in the absence of your non-test code changes. If your PR does not match these criteria, please specify why
```
Comment (RyanCavanaugh) [inline CONTRIBUTING.md:100]: Please revert. This isn't a typo.

Comment (Suryadevelops) [reply CONTRIBUTING.md]: Reverted

File: CONTRIBUTING.md (hunk 1 above)
Comment (Suryadevelops) [reply CONTRIBUTING.md]: Reverted

**Review by thetayloredman — APPROVED** [review]

**Review by sandersn — APPROVED** [review]

**Comment by Guada174** [comment]
npm install -g typescript

**Comment by Guada174** [comment]
> ``

**Pull request merged**